build.gradle

```diff
@@ -14,6 +14,9 @@ buildscript {
       }
     }
     gradlePluginPortal()
+    repositories {
```

Comment:
More context on the additional changes: https://github.com/mapbox/mapbox-navigation-android/pull/7301